scranShipper/src/RestaurantMenu/style.css

```diff
@@ -98,10 +102,10 @@
 
 @media screen and (max-width: 1000px) {
     .menuItem {
-        display: block;
+        /*display: block;*/
         margin: 1rem auto;
-        padding: 1rem;
-        width: 90%;
+        /*padding: 1rem;*/
+        /*width: 90%;*/
```

Comment:
again, this may have to be changed into bootstrap to save problems down the line